examples/analog_hamiltonian_simulation/04_Maximum_Independent_Sets_with_Analog_Hamiltonian_Simulation.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>No comma is needed for, "... as possible so that..."</p><p>A dash should be preferred to a hyphen in this case, "... same intersection—this is inefficient."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>